### ISSUE TITLE
Update docs for object import

### ISSUE THREAD

**Issue by kbrandwijk** [prisma/graphql-import#31]

**Comment by SpaceK33z** [comment]
Seems to be documented here now: https://oss.prisma.io/content/graphql-import/overview#import-from-strings

**Comment by kbrandwijk** [comment]
@SpaceK33z No, that doesn't even mention object import, as far as I can see. It's been a while, I'll look for an example.
